docs/work-avoidance.md

```diff
@@ -2,7 +2,11 @@
 
 > v2.9 and after
```

Comment:
did we ever work out why this is a v2.9 feature?

Reply:
No idea, unless PVCs were introduced then.